src/sentry/models/user.py

```diff
@@ -302,6 +302,16 @@ def get_orgs(self):
             ).values('organization'),
         )
 
+    def get_orgs_require_2fa(self):
+        from sentry.models import (Organization, OrganizationMember, OrganizationStatus)
+        return Organization.objects.filter(
+            flags=models.F('flags').bitor(Organization.flags.require_2fa),
+            status=OrganizationStatus.VISIBLE,
+            id__in=OrganizationMember.objects.filter(
```

Comment:
I'd just use the `member_set__user=self` syntax instead. It's a bit more clear imo and it produces a JOIN instead of a nested subquery.

But it yields the same results.